tests/epyccel/test_parallel_epyccel.py

```diff
@@ -73,3 +73,19 @@ def test_function(language):
     h_fast(x)
 
     assert np.array_equal(x, x_expected)
+
+#================================================================================
+@pytest.mark.parallel
+def test_module_3(language):
```

Comment:
```suggestion
def test_module_3(language_with_cuda):
```

Why is this test in this file? Does it run in parallel with MPI?